src/Jetpack_Usage.php

```diff
@@ -0,0 +1,161 @@
+<?php
+namespace Automattic\Jetpack\Tracking;
+
+use Automattic\Jetpack\Tracking;
+use Automattic\Jetpack\Connection\Manager as Connection_Manager;
+
+class Jetpack_Usage {
+
+	private $tracking;
+
+	function init() {
+		$this->tracking = new Tracking( 'jetpack' );
+
+		// For tracking stuff via js/ajax
+		add_action( 'admin_enqueue_scripts', array( $this->tracking, 'enqueue_tracks_scripts' ) );
+
+		add_action( 'jetpack_activate_module', array( $this, 'track_activate_module' ), 1, 1 );
+		add_action( 'jetpack_deactivate_module', array( $this, 'track_deactivate_module' ), 1, 1 );
+		add_action( 'jetpack_user_authorized', array( $this, 'track_user_linked' ) );
+		add_action( 'wp_login_failed', array( $this, 'track_failed_login_attempts' ) );
+
+		// Track that we've begun verifying the previously generated secret.
+		add_action( 'jetpack_verify_secrets_begin', array( $this, 'track_jetpack_verify_secrets_begin' ), 10, 2 );
+		add_action( 'jetpack_verify_secrets_success', array( $this, 'track_jetpack_verify_secrets_success' ), 10, 2 );
+		add_action( 'jetpack_verify_secrets_fail', array( $this, 'track_jetpack_verify_secrets_fail' ), 10, 3 );
+
+		// Universal ajax callback for all tracking events triggered via js
+		add_action( 'wp_ajax_jetpack_tracks', array( $this, 'jetpack_admin_ajax_tracks_callback' ) );
+
+		add_action( 'jetpack_verify_api_authorization_request_error_double_encode', array( $this, 'jetpack_verify_api_authorization_request_error_double_encode' ) );
+		add_action( 'jetpack_connection_register_fail', array( $this, 'jetpack_connection_register_fail' ) );
```

Comment:
We were missing the second argument here, so we need to make sure we're passing it to the callback.

```suggestion
		add_action( 'jetpack_connection_register_fail', array( $this, 'jetpack_connection_register_fail' ), 10, 2 );
```